runtime/runtime.go

```diff
@@ -633,7 +642,11 @@ func (r *interpreterRuntime) newInterpreter(
 		),
 		interpreter.WithUUIDHandler(func() (uuid uint64) {
 			wrapPanic(func() {
-				uuid = runtimeInterface.GenerateUUID()
+				var err error
+				uuid, err = runtimeInterface.GenerateUUID()
+				if err != nil {
+					panic(err)
+				}
```

Comment:
The `wrapPanic` should be only around the interface call:

```go
var err error
wrapPanic(func() {
	uuid, err = runtimeInterface.GenerateUUID()
})
if err != nil {
	panic(err)
}
```

Reply:
applied at 
19d823e